UPGRADE.md

```diff
@@ -0,0 +1,23 @@
+# Upgrading Phragile
+
+When upgrading Phragile to a newer version **always** run
+
+    composer install
+    php artisan migrate
+
+after updating the files in the root directory of your application.
+
+## From 1.x releases to 2.x.x
```

Comment:
Hmm, that's probably fine but I am wondering if it is not just safer (if no other reasons) to suggest to upgrade one version at a time, i.e. 1.0 to 1.1, 1.1 to 2.0.0 etc?
We have no idea (yet) what changes in the possible version 2.1 are going to be (who knows, maybe again some magic scripts will be needed to be run or whatnot), so maybe better just mention upgrade to 2.0.0 here?